tfhe/src/integer/server_key/crt/add_crt.rs

```diff
@@ -31,32 +32,34 @@ impl ServerKey {
         ct_left: &mut CrtCiphertext,
         ct_right: &mut CrtCiphertext,
     ) -> CrtCiphertext {
-        if !self.is_crt_add_possible(ct_left, ct_right) {
+        if self.is_crt_add_possible(ct_left, ct_right).is_err() {
             self.full_extract_message_assign(ct_left);
             self.full_extract_message_assign(ct_right);
         }
-        assert!(self.is_crt_add_possible(ct_left, ct_right));
+        assert!(self.is_crt_add_possible(ct_left, ct_right).is_ok());
```

Comment:
maybe unwrap to get the error message (same remark everywhere)

Reply:
Sure
If we could enable clippy lints on doc tests, we would probably have many such warnings

Reply:
there is an old issue in cargo for that, but if the doctests prove to be broken we'll likely add something to lint doctests